grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -184,6 +192,7 @@ class ClientBatchJob {
                 "${CLIENT_USERNAME}"     : credentialsBuilder.clientUsername,
                 "${SERVER_USERNAME}"     : credentialsBuilder.serverUsername,
                 "${SERVER_PASSWORD}"     : credentialsBuilder.serverPassword,
+                "${TRANSACTION_ID}"      : "${configsBuilder.transactionID}",
```

Comment:
A little more efficient:
```groovy
(TRANSACTION_ID)  : String.valueOf(configsBuilder.transactionID),
```

No need for parsing, building and converting GStrings back to Strings.